spec/generators/rspec/authentication/authentication_generator_spec.rb

```diff
@@ -19,6 +23,48 @@
       expect(File.exist?(file('spec/models/user_spec.rb'))).to be true
     end
 
+    it 'creates the request specs' do
+      run_generator
+
+      expect(File.exist?(file('spec/requests/sessions_spec.rb'))).to be true
+      expect(File.exist?(file('spec/requests/passwords_spec.rb'))).to be true
+    end
+
+    it 'configures the authentication support' do
+      # Create a minimal rails_helper.rb file that the generator can modify
+      FileUtils.mkdir_p(File.join(destination_root, 'spec'))
+      rails_helper_content = <<~CONTENT
+        # Rails.root.glob('spec/support/**/*.rb').sort_by(&:to_s).each { |f| require f }
+        RSpec.configure do |config|
+        end
+      CONTENT
+      File.write(File.join(destination_root, 'spec', 'rails_helper.rb'), rails_helper_content)
+
+      run_generator
+
+      expect(file('spec/rails_helper.rb')).to contain(
+        "  # Include authentication support module for request specs\n  config.include AuthenticationSupport, type: :request\n"
+      )
+      expect(file('spec/rails_helper.rb')).not_to contain(
+        "  # Rails.root.glob('spec/support/**/*.rb').sort_by(&:to_s).each { |f| require f }\n"
+      )
```

Comment:
Being pedantic, this only asserts the line was removed, I think prehaps match on the line without the #
```suggestion
      expect(file('spec/rails_helper.rb')).to contain(
        /^#{Regexp.escape("Rails.root.glob('spec/support/**/*.rb').sort_by(&:to_s).each { |f| require f }"}$/
      )
```